try{
console.log("hi");
throw new Error("something went wrong with the code");


}catch(e){
console.log(e);

}finally{

console.log("done");

}

console.log("hi");
